enterprise/metrics/src/main/java/org/neo4j/metrics/source/JvmMetrics.java

```diff
@@ -23,45 +23,33 @@
 import com.codahale.metrics.Metric;
 import com.codahale.metrics.MetricFilter;
 import com.codahale.metrics.MetricRegistry;
-import com.sun.management.GarbageCollectionNotificationInfo;
 
 import java.io.IOException;
 import java.lang.management.BufferPoolMXBean;
 import java.lang.management.GarbageCollectorMXBean;
 import java.lang.management.ManagementFactory;
 import java.lang.management.MemoryPoolMXBean;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.atomic.AtomicLong;
-import javax.management.ListenerNotFoundException;
-import javax.management.Notification;
-import javax.management.NotificationEmitter;
-import javax.management.NotificationListener;
-import javax.management.openmbean.CompositeData;
 
 import org.neo4j.kernel.configuration.Config;
-import org.neo4j.kernel.impl.logging.LogService;
 import org.neo4j.kernel.lifecycle.LifecycleAdapter;
 import org.neo4j.metrics.MetricsSettings;
 
 import static com.codahale.metrics.MetricRegistry.name;
-import static com.sun.management.GarbageCollectionNotificationInfo.GARBAGE_COLLECTION_NOTIFICATION;
 
 public class JvmMetrics extends LifecycleAdapter
 {
     private static final String NAME_PREFIX = "vm";
-    private static final String GC = name( NAME_PREFIX, "gc" );
+    private static final String GC_PREFIX = name( NAME_PREFIX, "gc" );
```

Comment:
I would assume we need to describe somewhere what those metrics show? Is it part of separate doc update commit?